### ISSUE TITLE
Missing package "aod" on EBRAINS

### ISSUE THREAD

**Issue by heplesser** [INM-6/multi-area-model#40]
I just tried to run the downscaled model on EBRAINS (Jülich Site) following the "Try it on EBRAINS" button. After a few cells, this fails with the message

```
Error in library("aod") : there is no package called ‘aod’
Execution halted
```

This seems closely related to #31, but now with the package missing on EBRAINS.

<img width="738" alt="Screenshot 2023-09-20 at 20 12 32" src="https://github.com/INM-6/multi-area-model/assets/1059644/a8e12e45-f9cb-480e-9fc4-89df82975566">

**Comment by heplesser** [comment]
Just for info, I was running the lastest version of MAM, 1.1.0, with git hash be361ff.

**Comment by heplesser** [comment]
As @diesmann just pointed out to me, the notebook actually runs through in spite of the error message. A very similar message occurs in Sec 2.3 of the notebook:

```
Initializing network from dictionary.
RAND_DATA_LABEL 9936
Error in library("aod") : there is no package called ‘aod’
Execution halted
No R installation or IndexError, taking hard-coded SLN fit parameters.
```

Here at least it says that it is using hard-coded parameters.

**Comment by didi-hou** [comment]
Hi, Hans. As Dennis replied in the Email, the packages aod is indeed missing at the moment, but it's an optional package which won't affect the running the following code. I've added the package and merged it to EBRAINS. I think the error will be fixed when the new EBRAINS is released this Friday. Thanks for checking ii. And @terhorstd The version number is also fixed yesterday as v1.1.0 and the relevant information is also updated in the package.py and spack.yaml. Everything works fine at the moment.
